pkg/notify/notify.go

```diff
@@ -56,45 +57,45 @@ func (r *Receiver) Notify(data *alertmanager.Data) (bool, error) {
 		// The set of JIRA status categories is fixed, this is a safe check to make.
 		if issue.Fields.Status.StatusCategory.Key != "done" {
 			// Issue is in a "to do" or "in progress" state, all done here.
-			log.V(1).Infof("Issue %s for %s is unresolved, nothing to do", issue.Key, issueLabel)
+			level.Debug(logger).Log("msg", "doing nothing, as issue is unresolved", "key", issue.Key, "label", issueLabel)
 			return false, nil
 		}
 		if r.conf.WontFixResolution != "" && issue.Fields.Resolution != nil &&
 			issue.Fields.Resolution.Name == r.conf.WontFixResolution {
 			// Issue is resolved as "Won't Fix" or equivalent, log a message just in case.
-			log.Infof("Issue %s for %s is resolved as %q, not reopening", issue.Key, issueLabel, issue.Fields.Resolution.Name)
+			level.Info(logger).Log("msg", "not reopening issue, as it was resolved", "key", issue.Key, "label", issueLabel, "resolution", issue.Fields.Resolution.Name)
 			return false, nil
 		}
 
 		resolutionTime := time.Time(issue.Fields.Resolutiondate)
 		if resolutionTime.Add(time.Duration(*r.conf.ReopenDuration)).After(time.Now()) {
-			log.Infof("Issue %s for %s was resolved on %s, reopening", issue.Key, issueLabel, resolutionTime.Format(time.RFC3339))
-			return r.reopen(issue.Key)
+			level.Info(logger).Log("msg", "reopening issue, as it was resolved after reopen duration, ", "key", issue.Key, "label", issueLabel, "reopenDuration", *r.conf.ReopenDuration, "resolutionTime", resolutionTime.Format(time.RFC3339))
+			return r.reopen(issue.Key, logger)
 		}
 	}
 
-	log.Infof("No issue matching %s found, creating new issue", issueLabel)
+	level.Info(logger).Log("msg", "opening a new issue, as no issue matching the label was found, ", "label", issueLabel)
```

Comment:
Not strictly related to your change, but since we now have a `reopen_duration` this log message should say `"no recent matching issue found, creating new issue".
```suggestion
	level.Info(logger).Log("msg", "no recent matching issue found, creating new issue", "label", issueLabel)
```